lib/dialects/abstract/query-generator.js

```diff
@@ -169,7 +171,14 @@ class QueryGenerator {
       }
     }
 
+    if (_.get(this, ['sequelize', 'options', 'dialectOptions', 'prependSearchPath']) || options.searchPath) {
```

Comment:
Use full path `sequelize.options.dialectOptions.prependSearchPath` in `_.get`

Reply:
I can change it, but it's more efficient in Lodash to use the array path form. If you use the string path form (e.g. `_.get(this, 'sequelize.options.dialectOptions.prependSearchPath')`) Lodash first converts it back to the array path form which currently entails a bunch of regex operations to parse the path. For a single call like this it probably doesn't matter, but do it many times and there's a big difference in performance.

Reply:
@gazoakley No need to change it then :)